src/state/workload.rs

```diff
@@ -242,17 +242,6 @@ impl Workload {
             service_account: self.service_account.clone(),
         }
     }
-    pub fn waypoint_svc_ip_address(&self) -> Result<Option<IpAddr>, WaypointError> {
```

Comment:
is there any linter in rust like unsed function?

Reply:
there is, i think maybe doesn't detect it due to it being public

Reply:
oha, thinked it in golang-way, lower-case function means private.